cpp/oneapi/dal/algo/connected_components/common.hpp

```diff
@@ -89,15 +89,15 @@ class descriptor : public detail::descriptor_base<Task> {
     using allocator_t = Allocator;
 
     explicit descriptor(Allocator allocator = std::allocator<char>()) {
-        _alloc = allocator;
+        alloc_ = allocator;
```

Comment:
Are these change actual?

Reply:
yes, @rlnx asks for changes in naming
probably not in connected component, but why not here too